docs/home/contribute/style-guide.md

```diff
@@ -399,6 +399,18 @@ considered new in a few months.
     <tr><td>The Federation feature provides ...</td><td>The new Federation feature provides ...</td></tr>
 </table>
 
+## Spell Checking
+
+Checking spelling before you propose changes to documents.  
+Do check it yourself as carefully as possible. Because false positives occur at a pretty high rate by automated tools.
+
+You can check typos with the help of [misspell](https://github.com/client9/misspell#install).
+
+Misspell has been used in Kubernetes and Community project:
+
+    ./hack/verify-spelling.sh
```

Comment:
I love this as an example of how to use the tool, with the exception that this repository doesn’t have a `hack` directory. Could we replace this example of how it’s used in Kubernetes with an example that you think is effective for use in writing/validating documentation in this repo?

Reply:
Please implement @steveperry-53's [feedback](https://github.com/kubernetes/website/pull/7691#issuecomment-371884158) here.